site/content/docs/developing/content-delivery.en.md

```diff
@@ -53,4 +53,24 @@ With both of these setups, Copilot will provision CloudFront to use the [SSL/TLS
 
 ## What is ingress restriction?
 
-You can restrict incoming traffic to come from a certain source. For CloudFront, Copilot uses an [AWS managed prefix list](https://docs.aws.amazon.com/vpc/latest/userguide/working-with-aws-managed-prefix-lists.html) to restrict allowed traffic to a set of CIDR IP addresses associated with CloudFront edge locations. When you specify `restrict_to.cdn: true`, your Public Load Balancer is no longer fully publicly accessible, and can only be accessed through the CloudFront distribution, guarding against security threats to your services.
\ No newline at end of file
+You can restrict incoming traffic to come from a certain source. For CloudFront, Copilot uses an [AWS managed prefix list](https://docs.aws.amazon.com/vpc/latest/userguide/working-with-aws-managed-prefix-lists.html) to restrict allowed traffic to a set of CIDR IP addresses associated with CloudFront edge locations. When you specify `restrict_to.cdn: true`, your Public Load Balancer is no longer fully publicly accessible, and can only be accessed through the CloudFront distribution, guarding against security threats to your services.
+
+## How do I use CloudFront to terminate TLS?
+
+You can optionally use CloudFront for TLS termination by configuring the env manifest as
+
+```yaml
+cdn:
+  tls_termination: true
+```
+
+And traffic from `CloudFront → Application Load Balancer (ALB) → ECS` will be HTTP only. This brings a very important benefit which is faster TLS termination, since the CloudFront edges are usually closer to the viewers.
+
+However, if HTTPS is enabled by either having the app domain or importing any certificates in the environment, you must turn off ALB http redirect by updating the service manifests as
```

Comment:
I wonder if we should use the `!!! attention ""` [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) here to highlight that they should do this first?

```
Before terminating TLS at the CDN level, we recommend:
1. First, disabling [HTTP to HTTPS redirection](fill-link-service-manifest) for your Load Balanced Web Services first. 
   ```yaml
   http:
      redirect: false
   ```
2. Run `copilot svc deploy`

Once all your services no longer, re-direct you can safely enable CloudFront TLS termination.
```